src/stirling/source_connectors/tcp_stats/BUILD.bazel

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018- The Pixie Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+load("//bazel:pl_build_system.bzl", "pl_cc_library", "pl_cc_test")
+
+package(default_visibility = ["//src/stirling:__subpackages__"])
+
+pl_cc_library(
+    name = "cc_library",
+    srcs = glob(
+        ["*.cc"],
+        exclude = [
+            "**/*_test.cc",
+        ],
+    ),
+    hdrs = glob(["*.h"]),
+    deps = [
+        "//src/stirling/bpf_tools:cc_library",
+        "//src/stirling/core:cc_library",
+        "//src/stirling/source_connectors/tcp_stats/bcc_bpf:tcpstats",
+        "//src/stirling/source_connectors/tcp_stats/bcc_bpf_intf:cc_library",
+    ],
+)
+
+pl_cc_test(
+    name = "tcp_stats_connector_test",
+    srcs = ["tcp_stats_connector_test.cc"],
+    tags = [
+        "requires_bpf",
+    ],
+    deps = [
+        ":cc_library",
+        "//src/stirling/testing:cc_library",
+    ],
+)
```

Comment:
This might be stale code. I think that the underlying test case was removed.

Reply:
Yes, updated build file